client/layout/hosting-dashboard/style.scss

```diff
@@ -45,6 +46,18 @@
 	flex: 1;
 }
 
+.hosting-dashboard-layout-column__container {
+	border-radius: 8px; /* stylelint-disable-line scales/radii */
+	overflow: hidden;
```

Comment:
Could you please let me know why we are setting `overflow: hidden;`. This seems to be break the scroll on A4A > Referrals: https://agencies.automattic.com/referrals/dashboard

@arthur791004